src/components/scenes/WalletListScene.js

```diff
@@ -10,7 +10,6 @@ import AntDesignIcon from 'react-native-vector-icons/AntDesign'
 import Ionicon from 'react-native-vector-icons/Ionicons'
 
 import credLogo from '../../assets/images/cred_logo.png'
-import iconImage from '../../assets/images/otp/OTP-badge_sm.png'
```

Comment:
Clearly, this line needs to come back using @amanalili's new `ModalImageIcon` component.